nototools/update_udhr_samples.py

```diff
@@ -24,17 +24,16 @@
 import collections
 import datetime
 import difflib
-import generate_website_data
 import os
 import re
 import shutil
-import unicode_data
-import urllib
+
+from six.moves.urllib.request import urlretrieve
```

Comment:
six is not part of the standard library, nor it is part of nototools setup.py 'install_requires' list of dependency.
If you really need to use this, you need to add it to that list.
Consider finding an alternative workaround instead.